internal/services/bot/bot_channels_registration_resource.go

```diff
@@ -146,6 +146,12 @@ func resourceBotChannelsRegistration() *pluginsdk.Resource {
 				ValidateFunc: validate.BotChannelRegistrationIconUrl,
 			},
 
+			"is_streaming_endpoint_enabled": {
```

Comment:
Might be better as just `streaming_endpoint_enabled`, WDYT?

Reply:
Sure. Updated